functional_tests/settings/python-m.yea

```diff
@@ -0,0 +1,13 @@
+id: 0.settings.python-m
+tag:
+    shard: service
+plugin:
+    - wandb
+command:
+    program: python -m module.lib
+assert:
+    - :wandb:runs_len: 1
+    - :op:contains:
+        - :wandb:runs[0][settings][program]
```

Comment:
im confused where did this come from?   settings isnt really known in the backend.

Shouldnt this be:
```
:wandb:runs[0][program]: -m module.lib
```

```

Reply:
this should run in the standard shard... we need to shard those tests, but your code isnt dependent on wandb-service